docs/CONTRIBUTING/Compiling.md

```diff
@@ -60,6 +60,17 @@ On windows cmake you need to specify the 'generator' type to the cmake invocatio
 cmake -DBUILD_ANDROID=On -DANDROID_ABI=armeabi-v7a -G "MSYS Makefiles" ..
 ```
 
+If using developing with multiple Android ABIs, naming your build folders as follows will allow renderdoc to find the right version to install automatically:
+
+| ABI         | Folder Name         |
+|-------------|---------------------|
+| armeabi-v7a | build-android-arm32 |
+| arm64-v8a   | build-android-arm64 |
```

Comment:
I don't think a table is the best way to convey this information. I think instead it would be better to have the above steps be duplicated under subheadings for arm32 and arm64 - that way it's easier for people to copy-paste the relevant instructions than have to parse out this data. There are only two options here so the duplication is minimal and worthwhile for ease of use.